requirements.txt

```diff
@@ -0,0 +1,44 @@
+alembic==0.8.6
```

Comment:
Is this additional file necessary? can't we just point people to `pip install -r requirements/prod.txt`?